internal/lsp/hover/hover_test.go

```diff
@@ -26,6 +27,18 @@ func TestCreateHoverContent(t *testing.T) {
 			ast.JSONFilter,
 			"testdata/hover/jsonfilter.md",
 		},
+		{
+			&ast.Builtin{
+				Name:        "foo.bar",
+				Description: "Description for Foo Bar",
+				Decl: types.NewFunction(
+					types.Args(),
+					types.Named("bar", types.A),
```

Comment:
Why aren't the return value name in the rendered markdown? Would be good to add a named arg too.

Reply:
Rebased to include this in a3e42835e2b300999a316508b5c8e7ebab6ccba4

Reply:
Nice, thanks!